### PR TITLE
#9087 Skip tests for pre-release e2e workflow on msedge/msedge-beta

### PR DESCRIPTION
**Pull request by mnholtz**
## What does this PR do?

- Part of #9087
- Temporarily skip the list of tests that are failing on the pre-release browser workflow for some reason. Follow-up ticket to investigate and fix is here: https://github.com/pixiebrix/pixiebrix-extension/issues/9125

### CODE REVIEW

File: end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts [hunk 1]
```diff
@@ -258,6 +263,13 @@ test("Add starter brick to mod", async ({
   });
 
   await test.step("Add Trigger starter brick to mod", async () => {
+    // FIXME: https://github.com/pixiebrix/pixiebrix-extension/issues/9125
+    test.skip(
```
Comment (fungairino) [inline end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts:267]: nit: we can use `test.fixme` instead which is functionally the same, but indicates we plan to fix it. Also the fixme comment is redundant - you can include the link in the additional description in the skip/fixme call.

https://playwright.dev/docs/api/class-test#test-fixme

File: end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts [hunk 2]
```diff
@@ -258,6 +263,13 @@ test("Add starter brick to mod", async ({
...
+      process.env.GITHUB_WORKFLOW === PRE_RELEASE_BROWSER_WORKFLOW_NAME &&
+        isMsEdge(chromiumChannel),
+      "Skipping test for MS Edge in pre-release workflow",
```
Comment (fungairino) [inline end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts:270]: Is it only failing in the pre-release workflow, but not in PR CI?

**Review by fungairino — APPROVED** [review]

File: end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts (hunk 2 above)
Comment (mnholtz) [reply end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts:270]: Yeah, based off of taking inventory of the CI runs in the last week or so. I'm not sure what is going on yet.

File: end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts (hunk 1 above)
Comment (mnholtz) [reply end-to-end-tests/tests/pageEditor/addStarterBrick.spec.ts:267]: Nice, thank you!

**Pull request merged**